algebird-core/src/main/scala/com/twitter/algebird/lsh/hashing/HashFamily.scala

```diff
@@ -0,0 +1,14 @@
+package com.twitter.algebird.lsh.hashing
+
+import com.twitter.algebird.lsh.vector.BaseLshVector
+
+trait Hasher {
```

Comment:
This abstraction doesn't seem right.  LSH isn't confined only to vectors of doubles (that's specific to Euclidean space).  Also Hasher as a mapping of something onto an Int isn't specific to LSH.  Likely we'd want to have `Hasher[-T] { def hash(t: T): Int }` at the algebird (or algebird.hashing) level.